app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt

```diff
@@ -157,21 +133,28 @@ class CallNotificationBuilder @Inject constructor(
         val userIdString = data.userId.toString()
         val conversationIdString = data.conversationId.toString()
         val channelId = NotificationConstants.getOutgoingChannelId(data.userId)
+        val person = Person.Builder().setName(data.conversationName).build()
 
-        return NotificationCompat.Builder(context, channelId)
-            .setPriority(NotificationCompat.PRIORITY_DEFAULT)
+        val notificationBuilder = NotificationCompat.Builder(context, channelId)
+        return notificationBuilder
+            .setPriority(NotificationCompat.PRIORITY_LOW)
             .setCategory(NotificationCompat.CATEGORY_CALL)
             .setSmallIcon(R.drawable.notification_icon_small)
             .setContentTitle(data.conversationName)
             .setContentText(context.getString(R.string.notification_outgoing_call_tap_to_return))
             .setAutoCancel(false)
             .setOngoing(true)
             .setSilent(true)
+            .setStyle(
+                CallStyle.forOngoingCall(
```

Comment:
If we remove `reloadIfNeeded` completely then can we also make outgoing and ongoing call notifications to be of the `CallStyle` so that they cannot be dismissed as well?

Reply:
Yes they are already

Reply:
Oh, sorry, I don't know why I missed that 😅